intlogger.go

```diff
@@ -123,6 +130,10 @@ func newLogger(opts *LoggerOptions) *intLogger {
 	return l
 }
 
+// offsetIntLogger is the stack frame offset in the call stack for the caller to
+// one of the Warn,Info,Log,etc methods.
+const offsetIntLogger = 3
```

Comment:
FYI this needs to be 6 in the case of log entries coming by way of the standard log package into a log.StandardWriter().

Issue #82.

Reply:
Ah yes, thank you for raising this problem!

This PR should make it easy to fix #82. I'll fix the merge conflicts in this branch and make that change.